recovery/README.md

```diff
@@ -9,40 +9,3 @@ the recovery tool.
 While that repo does offer prebuilt binary releases, we rebuild from scratch
 here so we can be sure about which TamaGo toolchain version is used, etc.
 
-## Build and Release Process
-
-A
-[Cloud Build trigger](https://cloud.google.com/build/docs/automating-builds/create-manage-triggers)
-is defined by a yaml config file. The Transparency.dev team invokes it manually
-when we want to publish a release.
-
-The pipeline includes two main steps: building and making available the recovery
-tool files, and writing the release metadata (Claimant Model Statement) to the
-firmware transparency log.
-
-1.  Cloud Build builds the recovery builder Docker image and copies the compiled
-    recovery imx file to a public Google Cloud Storage bucket.
-1.  Cloud Build runs the
-    [`manifest`](https://github.com/transparency-dev/armored-witness/tree/main/cmd/manifest)
-    tool to construct the Claimant Model Statement with arguments specific to
-    this release. It signs the Statement with the
-    [`sign`](https://github.com/transparency-dev/armored-witness/tree/main/cmd/sign)
-    tool and adds the resulting signed Statement as an entry to the public
-    firmware transparency log.
-
-TODO: add links for the GCS buckets once public.
-
-## Claimant Model
```

Comment:
Do we want to keep these claimant model docs around somewhere?

Reply:
Think it would be good to move them to `armored-witness`, where the Terraform is defined!